.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
```

Comment:
- It seems like you're repeatedly calling `echo -e` and `sed` on `RELEASE_NOTES`. This could be made more efficient by using `sed` only once with multiple `-e` options that specify all the necessary replacements. This would also improve the readability of the code.